spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -156,11 +162,40 @@ def get_parser():
         default=30,
         help="Degree of smoothing for centerline fitting. Only use with -centerline-algo {bspline, linear}."
     )
+    optional.add_argument(
+        '-pmj',
+        metavar=Metavar.file,
+        help="Ponto-Medullary Junction (PMJ) label file. "
+             "Example: pmj.nii.gz"
+    )
+    optional.add_argument(
+        '-distance',
+        type=float,
+        metavar=Metavar.float,
+        help="Distance (mm) from Ponto-Medullary Junction (PMJ) to compute CSA. "
+             "To be used with flag -pmj"
+    )
+    optional.add_argument(
+        '-extent',
+        type=float,
+        metavar=Metavar.float,
+        default=20,
+        help="Extent of the mask to average CSA (mm). "
+             "To be used with flag -pmj and -distance"
+    )
```

Comment:
Do we need both of these arguments?

Say the user might specify something like `-distance 50 -extent 30`. If this just means `-pmj-region 50:80`, then maybe it would be simpler/clearer to just have the user specify the region in a single argument.

Reply:
well it would actually be `-distance 50 -extent 30` --> `-pmj-region 35:65`, the extent is centered on the distance

I personnaly find it more intuitive to keep `-distance `and `-extent` since there is a default value of `-extent `and when using floats too. For example, I am using this feature in https://github.com/sct-pipeline/ukbiobank-spinalcord-csa, I decided `-distance 64`  since it is the mean distance from PMJ to C2-C3 disc, it is directly the input of `-distance`, so just one number to change if the mean changes. I would have to do the maths otherwise (±10, not very complicated I agree 😂)

maybe @jcohenadad has an opinion on this

Reply:
yes, I also think I prefer to have them separate, because of the default value for extent and also we don't need to parse it out (although it's trivial to do...). 

however we might want to consider using "pmj" as a prefix for these flags, eg: `-pmj-distance` and `-pmj-extent`

Reply:
Thank you for the clarification! Now I understand the intended usage much better (fix `-extent` and adjust `-distance` when fine-tuning).

Reply:
I changed the flags for `-pmj-distance` and `-pmj-extent` in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3478/commits/d3d6c3e3c6a26df276786056fe0e730867474df9